lib/dk/local.rb

```diff
@@ -0,0 +1,74 @@
+require 'scmd'
+
+module Dk; end
+module Dk::Local
+
+  class BaseCmd
+
+    def initialize(scmd_or_spy)
+      @scmd = scmd_or_spy
+    end
+
+    def cmd_str; @scmd.cmd_str; end
+
+    def run(input = nil);  @scmd.run(input);  self; end
+    def run!(input = nil); @scmd.run!(input); self; end
+
+    def stdout;   @scmd.stdout;   end
+    def stderr;   @scmd.stderr;   end
+    def success?; @scmd.success?; end
+    def to_s;     @scmd.to_s;     end
+
+    def output_lines
+      build_stdout_lines(self.stdout) + build_stderr_lines(self.stderr)
+    end
+
+    private
+
+    def build_stdout_lines(stdout)
+      build_output_lines('stdout', stdout)
+    end
+
+    def build_stderr_lines(stderr)
+      build_output_lines('stderr', stderr)
+    end
+
+    def build_output_lines(name, output)
+      output.strip.split("\n").map{ |line| OutputLine.new(name, line) }
```

Comment:
@kellyredding - Do you want to do `output.to_s` here to account for stdout/stderr being `nil`? I'm not sure if this is possible with scmd but it could be with an scmd spy if I manually write stdout/stderr with `nil`.